packages/shared/components/popups/StakingManager.svelte

```diff
@@ -90,23 +106,11 @@
         isPerformingAction = true
         $participatedAccountsMapPerSession.set($accountToParticipate?.id, false)
 
-        const isPartialStake = $partiallyStakedAccounts.find((psa) => psa.id === $accountToParticipate?.id) !== undefined
-
-        const _sync = async () => {
-            await getParticipationOverview()
-
-            showAppNotification({
```

Comment:
Do we still notify the user of a successful stake or unstake action?

Reply:
Yep, we do it [here](https://github.com/laumair/firefly/blob/feat/atomic-updates/packages/shared/lib/wallet.ts#L835-L843).